rootfs/usr/local/bin/xgo-build

```diff
@@ -99,7 +99,7 @@ else
 
   # Otherwise download the canonical import path (may fail, don't allow failures beyond)
   echo "Fetching main repository $1..."
-  GO111MODULE=off go get -v -d "$1"
+  GHQ_ROOT=$GOPATH_ROOT ghq get "$1"
```

Comment:
We need to ensure `ghq` is available in PATH before using it.

Reply:
Do I need to change something to achieve it? As ghq is installed in Dockerfile - shouldn't it be available in PATH at this point?

Reply:
This is just a nit to make sure it's available. I'm fine if we left this check.